bitnami/minio/templates/service.yaml

```diff
@@ -29,10 +29,18 @@ spec:
   {{- end }}
   ports:
     - name: minio
-      port: {{ .Values.service.port }}
-      targetPort: minio
+      port: {{ .Values.service.ports.api }}
+      targetPort: minio-api
       {{- if and (or (eq .Values.service.type "NodePort") (eq .Values.service.type "LoadBalancer")) (not (empty .Values.service.nodePort)) }}
-      nodePort: {{ .Values.service.nodePort }}
+      nodePort: {{ .Values.service.nodePorts.api }}
```

Comment:
I don't see this new variable neither in the changelog nor in the values.yaml file.